DEVELOPING.md

```diff
@@ -0,0 +1,67 @@
+# Developing
+
+There are a few useful things to know before diving into the codebase. This project depends on a few things being available 
+like a vulnerability database, which you might want to create manually instead of retrieving a released version.
```

Comment:
nit: I'd suggest no line breaks mid-paragraph for portability on all readers' perspective